omod/src/main/webapp/reports/manageReportDesigns.jsp

```diff
@@ -57,9 +57,9 @@
 				<c:forEach items="${reportDesigns}" var="design" varStatus="designStatus">
 					<tr>
 						<td width="20%" nowrap="true">
-							<a id="${design.uuid}DesignEditLink" href="#">${design.name}</a>
+							<a id="${design.uuid}DesignEditLink" href="#"><c:out value='${design.name}' /></a>
 						</td>
-						<td>${design.description}</td>
+						<td><c:out value="${design.description}"/></td>
```

Comment:
thanks @Parth59  do you have the screen shots of before and after?

Reply:
Yupp. I will attach those soon.

Reply:
Hi @HerbertYiga , I attached the images that you asked !